test/unit/v1.0.0/converter-v1-to-v2.test.js

```diff
@@ -698,6 +698,150 @@ describe('v1.0.0 to v2.0.0', function () {
             });
         });
 
+        it('should set mode to raw if dataMode is not set but all other data props are set', function (done) {
+            transformer.convert({
+                id: '84b2b626-d3a6-0f31-c7a0-47733c01d0c2',
+                name: 'null-dataMode',
+                order: ['4f65e265-dd38-0a67-71a5-d9dd50fa37a1'],
+                folders: [],
+                folders_order: [],
+                requests: [{
+                    id: '4f65e265-dd38-0a67-71a5-d9dd50fa37a1',
+                    headers: '',
+                    url: 'https://postman-echo.com/post',
+                    data: [{ key: 'foo', value: 'bar' }],
+                    rawModeData: 'RAW MODE HAVE HIGHEST PRECEDENCE',
+                    graphqlModeData: 'Am I a Joke To You?',
+                    method: 'POST',
+                    collectionId: '84b2b626-d3a6-0f31-c7a0-47733c01d0c2'
+                }]
+            }, options, function (err, converted) {
+                expect(err).to.not.be.ok;
+
+                // remove `undefined` properties for testing
+                converted = JSON.parse(JSON.stringify(converted));
+
+                expect(converted).to.eql({
+                    info: {
+                        _postman_id: '84b2b626-d3a6-0f31-c7a0-47733c01d0c2',
+                        name: 'null-dataMode',
+                        schema: 'https://schema.getpostman.com/json/collection/v2.0.0/collection.json'
+                    },
+                    item: [{
+                        _postman_id: '4f65e265-dd38-0a67-71a5-d9dd50fa37a1',
+                        name: '',
+                        request: {
+                            body: {
+                                mode: 'raw',
+                                raw: 'RAW MODE HAVE HIGHEST PRECEDENCE'
+                            },
+                            header: [],
+                            method: 'POST',
+                            url: 'https://postman-echo.com/post'
+                        },
+                        response: []
+                    }]
+                });
+                done();
+            });
+        });
+
+        it('should set mode to graphql if dataMode is not set but graphqlModeData is present', function (done) {
```

Comment:
group all graphql tests into their own `describe` block.

Reply:
All the different body types tests are grouped together already.